invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/MemberSearchBar.js

```diff
@@ -172,5 +172,6 @@ MembersSearchBar.propTypes = {
 };
 
 MembersSearchBar.defaultProps = {
+  searchType: "group",
```

Comment:
I am not sure about defaulting to group here...how is this used across the codebase? If we always pass a prop, then I will rather make the prop required than defining a default.